db/migrate/20190913125905_migrate_subjects.rb

```diff
@@ -0,0 +1,13 @@
+class MigrateSubjects < ActiveRecord::Migration[5.2]
+  def change
+    say_with_time 'populating/migrating course subjects' do
+      all_subject = Subject.all
+      all_course_includes_ucas_subjects = Course.includes(:ucas_subjects, :provider, :subjects)
```

Comment:
It looks to me like there were a couple of extra resources that can be loaded up here:

```suggestion
      all_course_includes_ucas_subjects = Course.includes(
        :ucas_subjects,
        :subjects,
        :enrichments,
        provider: :recruitment_cycle
      )
```

Reply:
`enrichments` ? seriously not a copy and paste craziness?

Reply:
with enrichments
```
== 20190913125905 MigrateSubjects: migrating ==================================
-- populating/migrating course subjects
   -> 213.1530s
== 20190913125905 MigrateSubjects: migrated (213.1531s) =======================
```


without enrichments

```
== 20190913125905 MigrateSubjects: migrating ==================================
-- populating/migrating course subjects
   -> 129.2781s
== 20190913125905 MigrateSubjects: migrated (129.2781s) =======================
```

Reply:
so droppping enrichment maybe it was needed cause of `save`